src/configuration/advanced.md

```diff
@@ -2,7 +2,7 @@
 title: Advanced
 ---
 
-The Advanced configuration settings can be found in **Stores** > **Configuration** > **Advanced**.
+The Advanced configuration settings can be found in **Stores** > **Settings** > **Configuration** > **Advanced**.
```

Comment:
Thanks for finding these nav issues. So you know, the standard format is to bold things you actually click on and _emphasis_ for headings such as _Settings_.

Reply:
@dobooth thanks for your notes.
Yes, I noticed that non-clickable elements are in italics.
Everything is ready for this request
Also, to follow a single style, I changed this on the rest of the pages so that it looks the same.
Thanks!